script/Ciel/Day 15/15_01A_CIEL15A_1.txt

```diff
@@ -949,9 +978,9 @@ I return to the mansion the following day.
 [sha:3c7da2420a0ed270d324d5d300e4485c423c856e]{
 -- Page 57, Offset 41556.
 -- 長く眠っていたせいか、ようやく帰ってきたというのに、これが初めての訪問のように思えた。
-It must be because I was asleep for so long, but...
- even though I'm finally coming back,
- it feels as though I'm seeing it for the first time.
+Maybe this is just because I was asleep for so long,
+ but even though I've finally returned home,
+ it almost feels like the first time.
```

Comment:
'feels like the first time' too vague in EN imo, hence why I elaborated on it before. You can do it shorter, but I think it needs the clarity.
> feels like it's my first time here.